package.json

```diff
@@ -16,7 +16,8 @@
   "dependencies": {
     "htmlparser2": "3.9.0",
     "memoizee": "0.3.10",
-    "promise": "7.1.1"
+    "promise": "7.1.1",
+    "scratch-render-webgl": "git+https://github.com/LLK/scratch-render-webgl.git"
```

Comment:
I think this should be a `devDependency`:
`"scratch-render": "git+https://git@github.com/LLK/scratch-render.git"`